### ISSUE TITLE
Transaction details not shown when "display in wallets list" not set

### ISSUE THREAD

**Issue by rafaelpac** [BlueWallet/BlueWallet#1966]
I have Android, Blue Wallet 5.6.1.

I have a Segwit P2SH wallet, it has a few UTXOs.

I had set the "transactions: display in wallets list" option to off.

I sent a transaction yesterday (I did it 1sat/b, so it is still not confirmed), and I noticed that I could not see the transaction details in Blue Wallet. I did not know from which UTXO it had been sent, but I managed to find the transaction in a block explorer searching by the destination address.

Now I just did another transaction, and again the transaction details were blank.

After a few tries, I tried setting the "transactions: display in wallets list" option to on, and the details are now shown (I then discovered that this new transaction used the unconfirmed output of the other one from yesterday, shouldn't it use one of the confirmed UTXOs I have in this same wallet?).

Now, if I set the "display" option again to off, I still see the transaction details, what makes me believe that the issue only occurs if the option is set to off during the transaction broadcast.

Sorry if I was not totally clear, I will be happy to help with any other info needed.

**Comment by Overtorment** [comment]
thanks for the report!
can you pls clarify

>>  I could not see the transaction details in Blue Wallet

what does it mean? screenshots would be helpful.



PS. BW tries to spend all available utxo, but we will add option "use only confirmed outputs", that's on our TODO lits

**Comment by rafaelpac** [comment]
![Capture+_2020-10-10-11-30-59](https://user-images.githubusercontent.com/6444444/95657718-51734300-0aec-11eb-9548-5e4e4d4c83c5.png)

This screenshot is not the real thing, because now, even if I set the option back to off, the details don't go away anymore. But it was just like that.

**Comment by rafaelpac** [comment]
![Capture+_2020-10-10-11-35-50](https://user-images.githubusercontent.com/6444444/95657795-d0687b80-0aec-11eb-86f3-19f1ff160ed2.png)
![Capture+_2020-10-10-11-35-39](https://user-images.githubusercontent.com/6444444/95657797-d199a880-0aec-11eb-8d76-334cdb217b5f.png)

Hi, I actually got something wrong before. I just tried again to set the option to off and the detail disappeared again. See above. I believe that before, when trying to reproduce the error, I hadn't saved the options after setting it to off.
